backend/course/crawler.py

```diff
@@ -36,8 +42,10 @@ def get_subareas(self, year, semester, area):
         return list(filter(lambda s: s['code'] != '', subareas))
 
     def get_courses(self, year, semester, area, subarea):
+        logging.debug('Crawling %s-%s %s-%s' % (str(year), str(semester), str(area), str(subarea)))
```

Comment:
```
'Crawling %s-%s %s-%s' % (
str(year),
self.semester_name[semester],
'(%s, %s)' % (area['code'], area['name']),
'(%s, %s)' % (subarea['code'], subarea['name'])
)
```
This can be more readable.